### PR TITLE
Document how to use vcpkg to install dependencies for the superbuild

### PR DESCRIPTION
**Pull request by traversaro**
@lucaTagliapietra using this procedure, I was able to compile the superbuild on the workstation your provided me. 

Fix https://github.com/robotology/robotology-superbuild/issues/42 .

### CODE REVIEW

**Comment by traversaro** [comment]
If you can signal the typos it would be helpful, thanks!

**Comment by traversaro** [comment]
friendly ping @diegoferigo

File: README.md [hunk 1]
```diff
@@ -245,17 +245,12 @@ For this reason if you plan to do use the robotology software with the Gazebo si
...
-( http://www.yarp.it/download.html#download_windows ) to install ACE, Eigen3 and Qt5
-and the binary installer of ICUB software ( http://wiki.icub.org/wiki/Windows:_installation_from_sources#Getting_iCub.27s_dependenceis ) to install Ipopt, OpenCV, SDL, Qt5 and GLUT.
-
-You can use those installers just to install YARP and iCub dependencies, as the latest version of YARP and ICUB will be installed by the superbuild itself.
 
-Software installed by the following [profile](#profile-cmake-options) or [dependencies](#dependencies-cmake-options) CMake options require specific enviromental variables to be set, as documented in options-specific documentation:
-* [`ROBOTOLOGY_ENABLE_IHMC`](#ihmc)
+As Windows does not have a widely used system [package manager](https://en.wikipedia.org/wiki/Package_manager) such as the one that are available on Linux or macOS, installing the system dependencies is slightly more compliated. However, we try to document every step necessary for the installation, but you find something that you don't understand in the documentation, please [open an issue](https://github.com/robotology/robotology-superbuild/issues/new).
```
Comment (diegoferigo) [inline README.md]: compliated -> complicated

File: README.md [hunk 2]
```diff
@@ -265,15 +260,56 @@ You can download the Git installer at http://msysgit.github.io/ .
 To install CMake you can use the official installer available at http://www.cmake.org/cmake/resources/software.html .
 It is recommended to install the latest version of CMake.
 
-### Superbuild
+#### Rapid Enviroment Editor
+While this tool is not strictly required, it is convenient to install the [Rapid Environment Editor](https://www.rapidee.com/en/download) to easily modify the value of the [environment variables](https://msdn.microsoft.com/en-us/library/windows/desktop/ms682653(v=vs.85).aspx) in Windows. 
+
+#### System Libraries
+The software in the superbuild depends on several libraries such as [Eigen](http://eigen.tuxfamily.org) and [Qt](https://www.qt.io/) that we assume are already available in your machine, as it would be too time intensive to build those libraries in the superbuild itself. There are two ways to install these libraries in your machine, either using the installer of YARP and iCub dependencies (the recommended way) or using [vcpkg](https://github.com/Microsoft/vcpkg) (not recommended, as it can takes several hours).
```
Comment (diegoferigo) [inline README.md]: can takes -> can take

File: README.md [hunk 3]
```diff
@@ -265,15 +260,56 @@ You can download the Git installer at http://msysgit.github.io/ .
...
+Use the `x86-windows` triplet only if you want to compile the superbuild using the 32-bit compiler. 
+The default way to use the libraries provided by vcpkg in CMake is to use the [vcpkg CMake toolchain](https://github.com/Microsoft/vcpkg/blob/master/docs/users/integration.md#cmake-toolchain-file-recommended-for-open-source-cmake-projects). 
+However, using a CMake toolchain is not currently supported by the superbuild, as the toolchain should be manually passed 
+when configuring each CMake-based subproject of the superbuild. 
+For this reason, to use the libraries installed by the superbuild, it is necessary to use the `CMAKE_PREFIX_PATH`,  `CMAKE_PROGRAM_PATH` and `PATH` environment variables. In particular, if your vcpkg is installed in `${VCPKG_ROOT}`, 
+you need to add `${VCPKG_ROOT}/installed/x64-windows` and `${VCPKG_ROOT}/installed/x64-windows/debug` to `CMAKE_PREFIX_PATH`,
+`${VCPKG_ROOT}/installed/x64-windows/tools` to `CMAKE_PROGRAM_PATH` and `${VCPKG_ROOT}/installed/x64-windows/bin` and `${VCPKG_ROOT}/installed/x64-windows/debug/bin` to `PATH`. 
+
+
+### Superbuild``
```
Comment (diegoferigo) [inline README.md]: Extra ``

**Comment by traversaro** [comment]
Thanks!

**Pull request merged**